### ISSUE TITLE
documentation shouldn't err or ""

### ISSUE THREAD

**Issue by bbatsov** [alexander-yakushev/compliment#35]
```clojure
cider.nrepl.middleware.complete> (jvm-complete/documentation "")
StringIndexOutOfBoundsException String index out of range: 0  java.lang.String.charAt (String.java:646)
```

Ideally this should return "" (what happens with unknown symbols).

**Comment by bbatsov** [comment]
More info - https://github.com/clojure-emacs/cider-nrepl/issues/294

**Comment by alexander-yakushev** [comment]
Agreed. Although I would expect CIDER middleware to catch all possible underlying exceptions and send the trace via its internal pipes.

**Comment by bbatsov** [comment]
That was just fixed in the middleware.

**Comment by alexander-yakushev** [comment]
OK. So should I still put a try block around that code? In that case I would just silently swallow exceptions which would make debugging more difficult.

Another thing to do is just to handle empty input to `documentation`.

**Comment by bbatsov** [comment]
I'd handle empty empty input. Seems like the reasonable thing to do.

**Comment by alexander-yakushev** [comment]
Fixed in e6040dfa8.
